src/main/src/main/java/org/geoserver/security/decorators/SecuredWebMapTileServer.java

```diff
@@ -71,11 +70,6 @@ public GetFeatureInfoResponse issueRequest(GetFeatureInfoRequest request) {
         return delegate.issueRequest(request);
     }
 
-    @Override
```

Comment:
Wait, why is this gone?

Reply:
Answering myself: because the constructor sets the capabilites field already, so there is no need to delegate. Not only that, due to GeoTools code changes, trying to do so causes NPEs during the class initialization.

Reply:
> Answering myself: because the constructor sets the capabilites field already, so there is no need to delegate. Not only that, due to GeoTools code changes, trying to do so causes NPEs during the class initialization.

@aaime That's exactly what I experienced this morning. Some tests failed because of that NPE.